pages/index.html

```diff
@@ -4,9 +4,72 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="/stylesheets/index.css">
+  <link rel="stylesheet" href="/stylesheets/style.css">
+  <title>Adiemon Fansite</title>
 </head>
+
 <body>
+  <header>
+    <nav>
+      <h1 id="oval-title">Welcome to our Adiemon Fansite</h1>
+      <h2>Adiemon Site Navigation</h2>
+      <ul>
+        <li id="visited">Home</li>
+        <li><a href="facts.html">Fun Facts</a></li>
+        <li><a href="gallery.html">Gallery</a></li>
+      </ul>
+    </nav>
+  </header>
+
   
+  
+  <main>
+    <section id="about-adiemon">
+      <h2>About Adiemon</h2>
+      <p>Welcome to our Adiemon fansite! This site is dedicated to exploring the world of Adiemon, a group of extraordinary creatures inspired by coding languages. Each Adiemon represents a different coding language, including Python, HTML, CSS, JavaScript, Flask, and React. Here, you'll find information, fun facts, and more about these unique digital creatures.</p>
+    </section>
+    <div class="section-container">
+      <section class="section-boxed">
```

Comment:
Class names like "section-container" and "section-boxed" aren't the most descriptive and could get confused in the css. Consider slightly more specific class names!